gulpfile.js

```diff
@@ -2,31 +2,46 @@ var gulp = require('gulp-param')(require('gulp'), process.argv),
     uglify = require('gulp-uglify'),
     rename = require('gulp-rename'),
     concat = require('gulp-concat'),
-    clean = require('gulp-clean');
+    clean = require('gulp-clean'),
+    pkg = require('./package.json');
     
     
-gulp.task('base', function() {
-  return gulp.src(['hand.base.js'])
-    .pipe(rename({suffix: '.min'}))
-    .pipe(uglify())
-    .pipe(gulp.dest('./'));
-});
+//gulp.task('base', function() {
+//  return gulp.src(['src/hand.base.js'])
+//    .pipe(rename({suffix: '.min'}))
+//    .pipe(uglify())
+//    .pipe(gulp.dest('dist/'));
+//});
+//
+//gulp.task('css', function() {
+//  return gulp.src(['src/hand.css.js'])
+//    .pipe(rename({suffix: '.min'}))
+//    .pipe(uglify())
+//    .pipe(gulp.dest('dist/'));
+//});
```

Comment:
It was duplicated from the default task. I haven't thinked that you are maybe using them in another context, I can amend my pull request to remove these changes, it's not an issue for the cdn. The only change needed is to have built and source files separated to distribute only built ones.